org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLFoldings.java

```diff
@@ -106,7 +106,7 @@ public TagInfo(int startLine, String tagName) {
 						}
 						int line = document.positionAt(scanner.getTokenOffset()).getLine();
 						int startLine = stackElement.startLine;
-						int endLine = line - 1;

```

Comment:
If `endLine` is just getting assigned `line`, and the value never changes, just rename `line` -> `endLine` and avoid the extra declaration.